tests/integration-tests/tests/common/utils.py

```diff
@@ -239,7 +239,9 @@ def reboot_head_node(cluster, remote_command_executor=None):
     logging.info(f"result.failed={result.failed}")
     logging.info(f"result.stdout={result.stdout}")
     wait_head_node_running(cluster)
-    time.sleep(120)  # Wait time is required for the head node to complete the reboot
+    # Wait time is required for the head node to complete the reboot.
+    # We observed that headnode in US isolated regions may take more time to reboot.
+    time.sleep(240 if "us-iso" in cluster.region else 120)
```

Comment:
should we just make it 240 for all?  WE've got a lot of us-iso caveats creeping into the codebase

Reply:
I think the peculiarities of us-iso regions should not have an impact on the rest of the regions we support.
Especially if this impact is relaxing constraints in integration tests.